Looks like we still need some files to be covered. From the above job:

```
2026-03-09T17:01:28.0119840Z Error: Detected file "Contents/Resources/app/extensions/positron-catalog-explorer/node_modules/snowflake-sdk/dist/lib/minicore/binaries/sf_mini_core_0.0.1.darwin-arm64.node" that's the same in both x64 and arm64 builds and not covered by the x64ArchFiles rule: "{*/kerberos.node,**/extensions/microsoft-authentication/dist/libmsalruntime.dylib,**/extensions/microsoft-authentication/dist/msal-node-runtime.node,**/node_modules/fsevents/fsevents.node,**/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node,**/node_modules/snowflake-sdk/dist/lib/minicore/binaries/*.darwin-x64.node}"
2026-03-09T17:01:28.0122200Z     at makeUniversalApp (/Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/node_modules/vscode-universal-bundler/dist/cjs/index.js:132:27)
2026-03-09T17:01:28.0123030Z     at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2026-03-09T17:01:28.0123790Z     at async origMain (file:///Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/darwin/create-universal-app.ts:222:2)
2026-03-09T17:01:28.0124740Z     at async main (file:///Users/ec2-user/actions-runner/_work/positron-builds/positron-builds/positron/build/darwin/create-universal-app.ts:198:3)
2026-03-09T17:01:28.0169820Z ##[error]Process completed with exit code 1.
```

--- a/build/darwin/create-universal-app.ts
+++ b/build/darwin/create-universal-app.ts
@@ -64,6 +64,8 @@ const stashPatterns = [
 	'**/node_modules/@napi-rs/canvas-darwin-arm64/skia.darwin-arm64.node',
 	// Exclusions from positron-assistant
 	'**/resources/copilot/**',  // Copilot language server binary
+	// Exclusions from Snowflake SQL client
+	'**/node_modules/snowflake-sdk/dist/lib/minicore/binaries/**',
 ];
 
 // Some generated files may end up being different in both distributions.
@@ -230,8 +232,7 @@ async function origMain(
 		// Add to the list of x64-only files
 		// **/node_modules/fsevents/fsevents.node
 		// **/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node
-		// **/node_modules/snowflake-sdk/dist/lib/minicore/binaries/*.darwin-x64.node
-		x64ArchFiles: '{*/kerberos.node,**/extensions/microsoft-authentication/dist/libmsalruntime.dylib,**/extensions/microsoft-authentication/dist/msal-node-runtime.node,**/node_modules/fsevents/fsevents.node,**/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node,**/node_modules/snowflake-sdk/dist/lib/minicore/binaries/*.darwin-x64.node}',
+		x64ArchFiles: '{*/kerberos.node,**/extensions/microsoft-authentication/dist/libmsalruntime.dylib,**/extensions/microsoft-authentication/dist/msal-node-runtime.node,**/node_modules/fsevents/fsevents.node,**/node_modules/@napi-rs/canvas-darwin-x64/skia.darwin-x64.node}',
 		// --- End Positron ---
 		filesToSkipComparison: (file: string) => {
 			for (const expected of filesToSkip) {

